scheduler/classes/WSRange.py

```diff
@@ -21,3 +22,108 @@ def getA1(self) -> str:
         """
 
         return self.start.getA1() + ":" + self.end.getA1()
+
+
+class CarBlock:
+    """Spreadsheet ranges corresponding to a car's output block
+
+        Attributes:
+            upper_left:
+            upper_right:
+            lower_right:
+            lower_left:
+            block_length:
```

Comment:
could you write a comment explaining what you mean by block length? Initially i thought it might be the length of the block you are writing (like upper_right - upper_left), but the way it is used makes it seem like its not that

Reply:
I was thinking length as in vertical length (like height) since we're writing top to bottom. Maybe block_height is a better name. I'll make the changes in the comments/docstring.

The block length changes car-to-car based on how many seats are available. It is equivalent to either upper row position minus and lower row position but the order in which the upper and lower positions are set I think prevent us from calculating it that way. Let me check, we might be able to eliminate block_length entirely.